src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/OffenderService.kt

```diff
@@ -433,11 +433,7 @@ class OffenderService(
     }
   }
 
-  fun getFilteredPrisonCaseNotesByNomsNumber(nomsNumber: String, getCas1SpecificNoteTypes: Boolean): CasResult<List<CaseNote>> {
-    val cas1PrisonNoteTypesToInclude = listOf(
-      "Alert", "Conduct & Behaviour", "Custodial Violence Management", "Negative Behaviours", "Enforcement", "Interventions / Keywork",
-      "Mental Health", "Drug Rehabilitation", "Social Care", "Positive Behaviour / Achievements", "Alcohol Treatment",
-    )
+  fun getPrisonCaseNotesByNomsNumber(nomsNumber: String): CasResult<List<CaseNote>> {
```

Comment:
I think you've lost the functionality the feature flag was enabling here. I can see `cas1PrisonNoteTypesToInclude` has been removed which presumably should be used by default for CAS1 now?

Reply:
I think this function should have been left as-is and the caller always pass in true for `getCas1SpecificNoteTypes` if the caller service is approved premises?